test/SILGen/accessibility_warnings.swift

```diff
@@ -56,15 +56,9 @@ extension PrivateStruct {
   public var publicVarExtension: Int { get { return 0 } set {} }
 }
 
-public extension PublicStruct {
-  // CHECK-DAG: sil @$S22accessibility_warnings12PublicStructV09extMemberC0yyF
-  public func extMemberPublic() {}
-  // CHECK-DAG: sil private @$S22accessibility_warnings12PublicStructV07extImplC033_5D2F2E026754A901C0FF90C404896D02LLyyF
-  private func extImplPublic() {}
-}
```

Comment:
Please don't remove this; just add the expected diagnostics. It's still checking the SIL-level linkage.

Reply:
Fixed.